packages/schematics/angular/utility/latest-versions/package.json

```
@@ -17,13 +17,13 @@
     "karma-jasmine": "~5.1.0",
     "karma": "~6.4.0",
     "less": "^4.2.0",
-    "ng-packagr": "^19.1.0-next.0",
+    "ng-packagr": "^19.1.0-next.2",
```

This needs to be `.0`. 

All framework and ng-packagr versions must be `^19.1.0-next.0` during the prerelease phase any other value is not allowed. 

```suggestion
    "ng-packagr": "^19.1.0-next.0",
```